Assets/UI/toppanel_CQUI.lua

```diff
@@ -9,9 +9,13 @@ BASE_CQUI_RefreshResources = RefreshResources;
 -- ===========================================================================
 local CQUI_showLuxury = true;
 
+function CQUI_OnSettingsInitialized()
```

Comment:
This looks good - we should check if we have the same issue for other settings

Reply:
I did a quick search and it seems we might have the same issue in a few other files. The ones below all register for the `LuaEvents.CQUI_SettingsUpdate` but don't register for `LuaEvents.CQUI_SettingsInitialized`
\Integrations\BTS\UI\tradeoverview.lua
\Integrations\BTS\UI\Choosers\traderoutechooser.lua
\Integrations\ML\Lenses\ModLens_Archaeologist.lua
\Integrations\ML\Lenses\ModLens_Builder.lua
\Integrations\ML\Lenses\ModLens_Scout.lua
The lens ones register for `Events.LoadScreenClose` and use the update function, and maybe that takes care of things.

I also saw one place where we register for `LuaEvents.CQUI_SettingsInitialized` but not for `LuaEvents.CQUI_SettingsUpdate`
\Integrations\ML\UI\minimappanel.lua
Over here we seem to register for `LuaEvents.CQUI_Option_ToggleYields` and I think that should take care of things there.

I think we should try and reproduce the error with the ones above before we file an issue.

Reply:
BTS and ML should be removed anyway, these are integrations. Here is just another argument why do that - even those integrations don't fully follow internal CQUI mechanisms.
I am not sure what error you wanna reproduce and where?